src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+ const cleanedUpString = sourceString
+    .split('\n')
+    .join('')
+    .trim()
+    .split(';')
+    .map(el => el.split(':'));
```

Comment:
[CODE KNOWLEDGE] - There's no need to make a copy before each operation. Methods like `split` and `map` do not mutate the original array, so you can chain these calls without using intermediate variables. Additionally, the `join('')` after splitting by newline characters is unnecessary and could potentially cause issues if there are styles that rely on newlines as part of their syntax (although not common in CSS). Consider chaining the methods directly and removing the `join('')` call.